pkg/templates/notificationtemplates/notification_generator_test.go

```diff
@@ -36,9 +36,18 @@ func TestGetNotificationTemplate(t *testing.T) {
 			require.NoError(t, err)
 			require.NotNil(t, template)
 			require.NotNil(t, template["userprovisioned"])
-			assert.Equal(t, template["userprovisioned"].Subject, "Notice: Your Red Hat CodeReady Toolchain account has been provisioned")
+			assert.Equal(t, "Notice: Your Red Hat CodeReady Toolchain account has been provisioned", template["userprovisioned"].Subject)
 			assert.Contains(t, template["userprovisioned"].Content, "You are receiving this email because you have an online <a href={{.RegistrationURL}}>Red Hat CodeReady Toolchain</a>")
 		})
+		t.Run("ensure cache is used", func(t *testing.T) {
+			// when
+			defer resetNotificationTemplateCache()
+
+			require.NotEmpty(t, UserProvisioned)
+			assert.Equal(t, "userprovisioned", UserProvisioned.Name)
+			assert.Equal(t, "Notice: Your Red Hat CodeReady Toolchain account has been provisioned", UserProvisioned.Subject)
+			assert.Contains(t, UserProvisioned.Content, "You are receiving this email because you have an online <a href={{.RegistrationURL}}>Red Hat CodeReady Toolchain</a>")
+		})
```

Comment:
Or just simply:
```suggestion
			assert.Equal(t, template["userprovisioned"], UserProvisioned)
		})
```

Reply:
fixed as part of: 43b2953aa3f7922169954600dd7d4cabfbe0a2ac